sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.CSharp9.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+
+int a = 0; // Compliant - FN
```

Comment:
this is a local variable in the `main` method - why is this an FN?

same for line 8

Reply:
My bad. It's not.